app/src/main/java/com/mobilecoronatracker/ui/about/CreditsListAdapter.kt

```diff
@@ -0,0 +1,29 @@
+package com.mobilecoronatracker.ui.about
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.mobilecoronatracker.databinding.ItemCreditsBinding
+import com.mobilecoronatracker.model.CreditsEntryModel
+
+class CreditsListAdapter : RecyclerView.Adapter<CreditsListAdapter.ViewHolder>() {
+    var credits = emptyList<CreditsEntryModel>()
+
+    inner class ViewHolder(private val binding: ItemCreditsBinding) :
+        RecyclerView.ViewHolder(binding.root) {
+        fun bind(item: CreditsEntryModel) {
+            binding.header.text = item.header
+            binding.body.text = item.body
```

Comment:
Is it working without `executePendingBindings`?  I think we should call it, to be sure that it will not overlap with old data. But maybe I have wrong info.

Reply:
Yes, you are right.